inc/wc-calypso-bridge-mailchimp-deactivate-hook.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Clear out MailChimp tables on deactivation
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+register_deactivation_hook( 'mailchimp-for-woocommerce/mailchimp-woocommerce.php', 'wc_calypso_bridge_clear_mailchimp_tables' );
+
+function wc_calypso_bridge_clear_mailchimp_tables() {
+	global $wpdb;
+	$log = new WC_Logger();
+
+	$mailchimp_tables = array(
+		"{$wpdb->prefix}queue",
+		"{$wpdb->prefix}failed_jobs",
+		"{$wpdb->prefix}mailchimp_carts",
+	);
+
+	foreach( $mailchimp_tables as $table ) {
```

Comment:
Optional. I almost always cast the expected array in a foreach explicitly as an array, e.g.

`foreach( (array) $mailchimp_tables as $table ) {`